lib/getstats.js

```diff
@@ -70,14 +70,15 @@ function _getStats(peerConnection, options) {
 function getActiveIceCandidatePairStats(peerConnection, options) {
   options = options || {};
 
-  if (typeof options.testForChrome !== 'undefined'
-    || typeof webkitRTCPeerConnection !== 'undefined') {
+  var browser = guessBrowser();
+
+  if (browser === 'chrome' || typeof options.testForChrome !== 'undefined') {
     return peerConnection.getStats().then(standardizeChromeActiveIceCandidatePairStats);
   }
-  if (typeof options.testForFirefox !== 'undefined'
-    || typeof mozRTCPeerConnection !== 'undefined') {
+  if (browser === 'firefox' || browser === 'edge' || typeof options.testForFirefox  !== 'undefined') {
```

Comment:
@syerrapragada How do you unit test for `edge` here without an `options.testForEdge` flag?